osh/hub/waiving/results_loader.py

```diff
@@ -45,10 +45,18 @@ def unpack_results(self):
         if task_has_results(self.target_task):
             logger.info("Results are already unpacked for task %s", self.target_task)
             return
-        else:
-            logger.debug('Unpacking %s', tb_path)
-            rex = ResultsExtractor(tb_path, output_dir=self.target_task_dir, unpack_in_temp=False)
-            rex.extract_tarball(self.exclude_dirs)
+
+        logger.debug('Unpacking %s', tb_path)
+        rex = ResultsExtractor(tb_path, output_dir=self.target_task_dir, unpack_in_temp=False)
+        rex.extract_tarball(self.exclude_dirs)
+
+        try:
+            with open(self.target_paths.get_txt_summary()) as f:
+                self.target_task.result = f.read().rstrip()
```

Comment:
Is there any size limit in the model?  If not we should enforce a size limit here, so that we do not store 1 GiB summary into the database if a scan goes wild.

Reply:
No, there is not: https://github.com/release-engineering/kobo/blob/9af65943b183e2904b47855cbc820b8dda2d5643/kobo/hub/models.py#L561

How many characters at most do you suggest to store, e.g. `5000`?

Reply:
We have valid summary files with ~20 KiB on the internal OSH instance, e.g. https://cov01.lab.eng.brq2.redhat.com/covscanhub/task/486761/log/kernel-5.14.0-427.5.1.el9_4/scan-results-all-summary.txt

I would set the limit to catch obvious mistakes only.  What about 256 KiB?

Reply:
Alternatively, we can keep it as a preview only.  If the summary is truncated, we can append a warning with a reference to full results, like we do for `stdout.log`.

Reply:
Addressed in 90a364bf995abf5ed3f16dbbb286e66252e9d163.